langchain/llms/base.py

```diff
@@ -286,6 +293,16 @@ def __call__(
             .text
         )
 
+    def predict(self, text: str, stop: Optional[List[str]] = None) -> str:
```

Comment:
```suggestion
    def predict(self, text: str, *, stop: Optional[Sequence[str]] = None) -> str:
```

The List -> Sequence change at this stage may involve having to propagate the type change down the call stack. If so, either we do it at least partially and then invoke `list` when needed, or not do for now.

Reply:
will do partially (for new methods) for now